cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -925,6 +925,12 @@ function Configure-GcePdTools {
 '$modulePath = "K8S_DIR\GetGcePdName.dll"
 Unblock-File $modulePath
 Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
+
+  if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
+    MustDownload-File -OutFile ${env:K8S_DIR}\diskutil.exe `
+      -URLs "https://ddebroywin1.s3-us-west-2.amazonaws.com/diskutil.exe"
```

Comment:
Can you add a comment with a link to the current source code for this? I'm guessing it's in Deep's github somewhere :)

Reply:
it is available in oss now https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c